content/docs/04-clusters/06-cluster-management/05-backup-restore.md

```diff
@@ -11,320 +11,493 @@ import WarningBox from 'shared/components/WarningBox';
 import InfoBox from 'shared/components/InfoBox';
 import PointsOfInterest from 'shared/components/common/PointOfInterest';
 
-# Overview
 
-Palette provides two ways to back up and restore Kubernetes clusters:
+# Cluster Backup and Restore
 
-* Cluster Backup and Restore for a single cluster which is managed from within the cluster.
-* [Workspace](/workspace/workload-features#workspaceoperator) Backup and Restore for multiple clusters managed from workspaces.
+A backup is a persistent state of Kubernetes clusters, including the native objects such as Pods, DaemonSets, Services, and persistent volumes. A backup allows you to save the current state of a cluster and restore it at a later point in time if needed. You can restore a backup to the same or a different cluster. Internally, Palette leverages an open-source tool called *Velero* to provide backup and restore capabilities. 
```

Comment:
```suggestion
A backup is a persistent state of Kubernetes resources, ranging from objects such as Pods, DaemonSets, Services, to persistent volumes. A backup allows you to save the current state of a cluster and restore it at a later point in time if needed. You can restore a backup to the same or a different cluster. 
```
The first sentence had improve use of plurals - "the native objects", "the" is incorrect in this usage. It would work if you remove the article, so it stated "including native objects" but the article makes it specific and that's where the plural crashes.

A backup can be specific to a set of resources, not necessarily the cluster itself, that's why the sentence is changed in the suggestion.